supplementary_style_guide/introduction/about-guide.adoc

```diff
@@ -4,6 +4,11 @@ This guide provides style guidelines and preferred term usage for link:https://w
 
 The Red Hat Customer Content Services team created this guide as part of its responsibility for producing product documentation for Red Hat customers. Upstream communities who want to align more closely with the standards used by Red Hat product documentation can also use this guide. 
 
+[NOTE]
+====
+Both the link:https://www.ibm.com/docs/en/ibm-style[IBM Style Guide] and the link:https://github.com/redhat-documentation/supplementary-style-guide/issues[Red Hat supplementary style guide] are the source of truth for output and writing style guidance that is not specific to Modular Documentation guidelines. The link:https://redhat-documentation.github.io/modular-docs/[Modular Documentation Reference Guide] and templates are the source of truth for all things connected to Modular Documentation, including implementing those guidelines in Asciidoc. The link:https://redhat-documentation.github.io/asciidoc-markup-conventions/[AsciiDoc Mark-up Quick Reference] is the source of truth for guidance specific to writing in Asciidoc.
+====
+
```

Comment:
@mportman12 Thank you so much for creating this PR! The text looks good to me. But I would move this from being a note to a full fledged section with a subheading under How to use this guide, or even as a heading 2 either before or after How to use this guide.

Reply:
@stoobie Moved the paragraph to a new section. I also removed the link to the SSG because it's irrelevant - the reader is already in the SSG.